### ISSUE TITLE
planitineraire.fr

### ISSUE THREAD

**Issue by gendy54** [osm-fr/attributions#505]
### Ce signalement concerne…

- https://planitineraire.fr/ 

### Présence de l'attribution

❌ Pas d'attribution

### (Facultatif) Quel est le fond de carte ?

openstreetmap.org

### Prise de contact

 Je l'ai contacté

### Informations supplémentaires

- Responsable contacté le : 
  - Responsable contacté par (courriel, Twitter…) : message de contact https://planitineraire.fr/contact
- Lié au signalement : #

**Comment by vinber** [comment]
no, if you change tiles there's no attribution and it's osm.org

<img width="1496" height="766" alt="Image" src="https://github.com/user-attachments/assets/0b2f591b-bdfc-4f9d-a88b-e58fc9f3062c" />
